logger/logger.go

```diff
@@ -153,3 +162,14 @@ func SimpleSetup() error {
 	}
 	return err
 }
+
+var procCmdline = "/proc/cmdline"
+
+func debugEnabledOnKernelCmdline() bool {
+	buf, err := ioutil.ReadFile(procCmdline)
+	if err != nil {
+		return false
+	}
+	l := strings.Split(string(buf), " ")
+	return strutil.ListContains(l, "snapd.debug=1")
```

Comment:
it would be great if we could move proc/cmdline parsing into a common package that can be used multiple places because this is now the second place where we do parsing of proc/cmdline (the first being boot, but that's only a recent transition, it used to be more places like initramfs-mounts, and boot and ...), but we shouldn't import boot here because we already import logging into boot. 

Maybe kernelutil.CmdlineContains("snapd.debug=1") or kernelutil.CmdlineGetVal("snapd.debug") etc.

anyways not for this PR

Reply:
if we introduce kernelutil there's a couple of things that could move there from osutil.